data/entities.json5

```diff
@@ -2900,8 +2900,9 @@
     "domains": ["*.cdnplanet.com"]
   },
   {
-    "name": "CDN.net",
+    "name": "InAuth",
     "categories": ["utility"],
+    "homepage": "https://www.inauth.com/"
```

Comment:
Anything we can link to for posterity that shows InAuth's connection to this domain? (WHOIS, article, existing site source, docs, github search, anything works :) )

The owner of the domain is fairly obscured 😄

Reply:
Ah you predicted my question all ready! Would you be willing to throw a screenshot of those query results on this PR and we'll call it a day? :D

Reply:
And just to validate that those URLs are indeed for InAuth, here's the current content of the scripts

![image](https://user-images.githubusercontent.com/2301202/57894184-7c7c3900-780b-11e9-823c-31d28782ec24.png)

Reply:
haha, just commented the same thing